README.md

```diff
@@ -1,81 +1,99 @@
+[JIRA](https://simp-project.atlassian.net/).
 
 Please read our [Contribution Guide](https://simp.readthedocs.io/en/stable/contributors_guide/index.html).
 
-This module is optimally designed for use within a larger SIMP ecosystem, but it can be used independently:
+This module is optimally designed for use within a larger SIMP ecosystem, but
+it can be used independently:
 
-* When included within the SIMP ecosystem, security compliance settings will be managed from the Puppet server.
+* When included within the SIMP ecosystem, security compliance settings will be
+  managed from the Puppet server.
 
 ## Module Description
 
-This module installs and configures freeradius. Its main purpose is to integrate freeradius
-with an existing LDAP server. It includes manifests that configure the ldap module and create
-a virtual server (site) that configures freeradius to listen on all available interfaces and
-authenticate via LDAP.
+This module installs and configures freeradius. Its main purpose is to
+integrate freeradius with an existing LDAP server. It includes manifests that
+creates a virtual server (site) that configures freeradius to listen on all
+available interfaces and authenticate via LDAP.
+
+See [REFERENCE.md](REFERENCE.md) for more details.
 
 ## Beginning with freeradius
 
-Before installing pupmod-simp-freeradius make sure to read the [freeradius documentation](http://freeradius.org/documentation)
+Before using pupmod-simp-freeradius make sure to read the
+[freeradius documentation](http://freeradius.org/documentation)
 
-Much of the freeradius documentation is in the default configuration files, some of which get overwritten by this module.  It could be helpful to extract and store these files
-in a seperate location using the command:
+Much of the freeradius documentation is in the default configuration files,
+some of which get overwritten by this module.  It could be helpful to extract
+and store these files in a separate location using the command:
 
+```shell
 rpm2cpio <free radius rpm> | cpio -idmv
+```
 
 ## Setup
 
-* Ensure the freeradius, freeradius-ldap and freeradius-utils packages are available.
+* Ensure the freeradius, freeradius-ldap and freeradius-utils packages are
+  available to your package manager.
 
 
 ### Defaults
 
-* Configuration directory: /etc/raddb
-* Log Directory: /var/log/freeradius
-* Ldap Bind user: bind_dn
-* Rsync: false
+* Configuration directory: `/etc/raddb`
+* Log Directory: `/var/log/freeradius`
+* Ldap Bind user: `bind_dn`
+* Rsync: `false`
 
 ### Set up Radius Server to use LDAP
 
-This basic setup will configure Radius to listen on all interfaces and authenticate
-using LDAP.
+This basic setup will configure RADIUS to listen on all interfaces and
+authenticate using LDAP.
 
 #### Install freeradius and the LDAP module and site configuration.
 
-Add the following to hiera for the Radius server:
+Add the following to Hiera for the Radius server:
 
 ```yaml
```

Comment:
In another PR you called out how the `classes` key was for a SIMP system.  Shouldn't you do that here, as well?